README.md

```diff
@@ -176,12 +176,17 @@ Instead we need to modify layout A's template, and mark the ``description`` anno
 Running Portia
 --------------
 
-Checkout the repository:
+### Dependencies ###
+You might need to install the following packages
```

Comment:
Mention this is for Ubuntu or Debian. OSX and Windows users can install the packages as binaries from pip.

"When running Portia on Ubuntu or Debian systems you may need to install the following dependencies:"